### DISCUSSION TITLE
world space animated vertices?

### DISCUSSION THREAD

**Discussion by ThomasCartier** [FyroxEngine/Fyrox#579] [Q&A] [answered]
Hi,

What would be the best approach to extract the world space vertices of an animated model at every step? It's from a humanoid.

Reading the doc, I may have missed the info.
Thanks,

**Comment by mrDIMAS** [comment, marked as answer]
Hi! I've added an example here - https://fyrox-book.github.io/fyrox/scene/mesh_node.html#data-buffers . What is your use case btw?

**Reply by ThomasCartier** [reply]
Thanks! Yes the use case is the following: I have a sequence in the game that rewards the player depending on some positions that are randomly pregenerated in a mesh. The vertices of this mesh are the locations to reach, the triggers; These locations don't follow a physics logic. The goal is to delegate to the physic engine these calculations at runtime.
:smile:
